components/authz-service/storage/postgres/migration/sql/47_staged_rules_tables.up.sql

```diff
@@ -0,0 +1,94 @@
+BEGIN;
+
+CREATE TABLE iam_staged_projects (
+  db_id SERIAL,
+  id TEXT PRIMARY KEY,
+  name TEXT NOT NULL,
+  type iam_policy_type NOT NULL DEFAULT 'custom',
+  projects TEXT[],
+  deleted BOOLEAN,
+  updated_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
+  created_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
+);
+
+CREATE TABLE iam_staged_project_rules (
+  db_id SERIAL PRIMARY KEY,
+  id TEXT NOT NULL UNIQUE,
+  project_id TEXT REFERENCES iam_staged_projects ON DELETE CASCADE,
+  name TEXT NOT NULL,
+  type TEXT NOT NULL,
+  deleted BOOLEAN,
+  updated_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
+  created_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
```

Comment:
I remember us deciding not to have these timestamps when we were kicking off the V2 work (we had them in V1 inconsistently) but I can't really remember the reason or conversation 😓 I guess my feeling is that if we add them here we should add them everywhere for IAM.